ale_linters/markdown/textlint.vim

```diff
@@ -0,0 +1,9 @@
+" Author: tokida https://rouger.info
+" Description: textlint, a proofreading tool (https://textlint.github.io/)
+
+call ale#linter#Define('markdown', {
+\   'name': 'textlint',
+\   'executable': 'textlint',
+\   'command': 'if [ -f .textlintrc ]; then textlint -f json %t ; else echo "[{\"filePath\":\"%t\",\"messages\":[]}]"; fi',
```

Comment:
There are two problems here.

1. This code will only work on some Unix variants.
2. You're only looking for a file in what _might_ be the current working directory, or which could be somewhere else entirely.

Try using `ale#path#FindNearestFile` in a command callback instead. See the `cppcheck.vim` file for an example, which is the first one I found.

Reply:
You don't need the part where you echo some default JSON. Return `''` from the callback instead to skip running the linter.